### ISSUE TITLE
putTemplate() doesn't do anything. [v5.x]

### ISSUE THREAD

**Issue by 8lall0** [elastic/elasticsearch-php#815]
### Summary of problem or feature request

The putTemplate() method simply doesn't do anything. Maybe i'm doing something wrong, but documentation is basically absent (or i didn't find it)

### Code snippet of problem

```php
$data = [
			'name' => 'testTemplate',
			'body' => [
				'template' => 'testTemplate-*',
				'mappings' => [
					'foo' => 'string',
					'bar' => 'string',
				],
			],
		];

$this->client = ClientBuilder::create()
		->allowBadJSONSerialization()
		->setHosts($host)
		->build();
$this->client->putTemplate($data);
```

### System details

* Wamp64 on Windows 10
* PHP 5.6
* ES-PHP 5.x
* ElasticSearch 5.6.8

**Comment by feydan** [comment]
I added comments here: https://github.com/elastic/elasticsearch-php/pull/817.

TL;DR, try `$this->client->indices()->putTemplate($data)`

**Comment by 8lall0** [comment]
Nice and neat, thanks!
